templates/data/mongodb/managed.html

```diff
@@ -0,0 +1,466 @@
+            </p>
+          </div>
+        </div>
+        <hr class="is-muted" />
+        <div class="row">
+          <div class="col-3 col-medium-3">
+            <h3 class="p-heading--5">Secure and compliant</h3>
+          </div>
+          <div class="col-6 col-medium-3">
+            <p>
+              Canonical ensures that MongoDB stays secure by providing patches for Common Vulnerabilities Exposure (CVE). In addition, Charmed MongoDB includes security features such as authentication and authorisation, Transport Layer Security (TLS), user management and password rotation.
+            </p>
+          </div>
+        </div>
+        <hr class="is-muted" />
+        <div class="row">
+          <div class="col-3 col-medium-3">
+            <h3 class="p-heading--5">Innovate at scale with your data</h3>
+          </div>
+          <div class="col-6 col-medium-3">
+            <p>
+              Canonical's managed services for MongoDB scale efficiently &mdash; so your MongoDB service remains stable and exceptionally cost effective as your business grows.
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="p-section--shallow">
+      <div class="row">
+        <hr />
+        <div class="col-6 col-medium-6">
+          <h2>Our process</h2>
+        </div>
+        <div class="col-6 col-medium-6">
+          <p>
+            Our Managed Solutions process is easy and intuitive. You'll initially be consulted by our specialists to design the best managed solution for your needs. Afterwards, it's all on us:
+          </p>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-9 col-medium-6 col-start-large-4">
+        <hr class="is-muted" />
+        <div class="row">
+          <div class="col-3 col-medium-3">
+            <h3 class="p-heading--5">1. Deployment</h3>
+            <div class="p-image-wrapper u-hide--small" style="padding:10%;">
+              {{ image(url="https://assets.ubuntu.com/v1/f887a7db-deploy+fast.svg",
+                            alt="",
+                            width="263",
+                            height="150",
+                            hi_def=True,
+                            attrs={"class": "p-image-container__image"},
+                            loading="lazy") | safe
+              }}
+            </div>
+          </div>
+          <div class="col-6 col-medium-3">
+            <p>Specialised Canonical engineers deploy a database sustainably and efficiently on any cloud.</p>
+          </div>
+        </div>
+        <hr class="is-muted" />
+        <div class="row">
+          <div class="col-3 col-medium-3">
+            <h3 class="p-heading--5">2. Management</h3>
+            <div class="p-image-wrapper u-hide--small" style="padding:10%;">
+              {{ image(url="https://assets.ubuntu.com/v1/3089fff7-one+platform.svg",
```

Comment:
The images should be below the paragraph text rather than the headings, so maybe in the next `col-6`